README.md

```diff
@@ -207,6 +207,23 @@ as well. A few examples:
 ```
 Again, modules can be provided as already loaded ones or as string.
 
+### Creating `pip` requirements files
+
+To make a `pip` requirements file, use the `Report.listing()` method in the
+following manner:
+
+```py
+import scooby
+report = scooby.Report()
+with open('requirements.txt', 'w') as f:
+    f.write(report.listing())
+```
```

Comment:
I left a comment in the main PR-thread. I would implement this within `report.listing` itself, not make the user a `with open...`.